README.md

```diff
@@ -0,0 +1,89 @@
+# Atlas Basic Terraform Module
+
+This Terraform module creates a basic setup on [MongoDB Atlas] [atlas]. 
+
+It creates the following resources:
+
+- A MongoDB Atlas project.
+- One or more database users.
+- Zero or more IP addresses.
+- Zero or more CIDR blocks.
+- A MongoDB Atlas cluster provisioned with AWS, GCP or AZURE provider. It can also be a free tier (TENANT) cluster.
```

Comment:
should we be explicit that it is a replica set?

Reply:
It is later explained in the considerations section, I'm not sure if it would be too redundant